app/editor/src/features/content/form/ContentClipForm.tsx

```diff
@@ -222,26 +232,10 @@ export const ContentClipForm: React.FC<IContentClipFormProps> = ({
           </div>
         </Col>
       </Row>
-      <Row justifyContent="center" className={!streamUrl ? 'hidden' : ''}>
+      <Row justifyContent="center" className={!stream ? 'hidden' : ''}>
         <Col className="video">
           <Row>
             <video ref={videoRef} controls>
-              <source src={streamUrl} type="audio/m4a" />
```

Comment:
Doesn't work correctly in Chrome.  However, this does highlight a challenge when we want to support different media types in the future.